catkin_tools/verbs/catkin_path/cli.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2014 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import os
+
+from catkin_pkg.packages import find_packages
+from catkin_pkg.package import InvalidPackage
```

Comment:
These two imports are unused.

I use a flake8 plugin which would have told you about these unused imports and that you were missing the `import sys`. They have one for most editors, I highly recommend it.